build/postgres/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apt-get update && \
         libpq-dev \
         libxml2-dev \
         pkg-config \
+        postgresql-client-${POSTGRES_VERSION} \
```

Comment:
When I tried building the ``9.5`` postgres image ``9.5.20`` was tagged as ``9.5``:
````
/tmp/musicbrainz-docker/build/postgres $ docker pull postgres:9.5.20
9.5: Pulling from library/postgres
619014d83c02: Already exists
7ec0fe6664f6: Already exists
9ca7ba8f7764: Already exists
9e1155d037e2: Already exists
febcfb7f8870: Already exists
8c78c79412b5: Already exists
5a35744405c5: Already exists
27717922e067: Already exists
48127670a602: Pull complete
6aedb706f269: Pull complete
158b65b6f8f1: Pull complete
7e3410bd4805: Pull complete
2c187ebaa355: Pull complete
a93e50cc434a: Pull complete
Digest: sha256:40cf77833a45615aa06e5073669f7083f1b1cdd350bae1724425e3d8333389dc
Status: Downloaded newer image for postgres:9.5
docker.io/library/postgres:9.5

/tmp/musicbrainz-docker/build/postgres $ git br
  master
* mbvm-38-dev

/tmp/musicbrainz-docker/build/postgres $ docker build .
[+] Building 12.2s (9/15)
 => [internal] load build definition from Dockerfile                                                                                                                                                     0.2s
 => => transferring dockerfile: 1.17kB                                                                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                                                                        0.1s
 => => transferring context: 2B                                                                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/postgres:9.5.20                                                                                                                                       0.0s
 => [internal] load build context                                                                                                                                                                        0.1s
 => => transferring context: 35B                                                                                                                                                                         0.0s
 => [1/12] FROM docker.io/library/postgres:9.5.20                                                                                                                                                        0.0s
 => CACHED [2/12] RUN apt-get update &&     apt-get -y -q install         build-essential         git-core         libdb-dev         libexpat1-dev         libicu-dev         libpq-dev         libxml2  0.0s
 => [3/12] RUN git clone https://github.com/metabrainz/postgresql-musicbrainz-unaccent.git /tmp/postgresql-musicbrainz-unaccent &&     git clone https://github.com/metabrainz/postgresql-musicbrainz-  10.4s
 => [4/12] WORKDIR /tmp/postgresql-musicbrainz-unaccent                                                                                                                                                  0.1s
 => ERROR [5/12] RUN make && make install                                                                                                                                                                1.4s
------
 > [5/12] RUN make && make install:
#8 1.258 Makefile:25: /usr/lib/postgresql/12/lib/pgxs/src/makefiles/pgxs.mk: No such file or directory
#8 1.259 make: *** No rule to make target '/usr/lib/postgresql/12/lib/pgxs/src/makefiles/pgxs.mk'.  Stop.
````

But when I build specifically against ``9.5.21`` the bug does not happen.
I suggest we just track postgres ``9.5.21`` for now.

Reply:
Thanks, so this is a new bug due to upstream. I can reproduce it after clearing caches, but pulling 9.5.21 doesn’t work for me either.

Reply:
So what do we do about this? It seems that the container is partially pg 12 and not 9.5 which is kinda confusing and not good at all.

Reply:
Issue was installed version of `libpq5` = 12. I pushed https://github.com/metabrainz/musicbrainz-docker/commit/af484dfc830563a917d2782b11f35451ed2e23af fix to [mbvm-38-dev](https://github.com/metabrainz/musicbrainz-docker/tree/mbvm-38-dev).

```diff
@@ -8,13 +8,14 @@ ARG POSTGRES_VERSION
 
 # Update the Ubuntu and PostgreSQL repository indexes
 RUN apt-get update && \
-    apt-get -y -q install \
+    apt-get -y -q install --allow-downgrades \
         build-essential \
         git-core \
         libdb-dev \
         libexpat1-dev \
         libicu-dev \
-        libpq-dev \
+        libpq5=${POSTGRES_VERSION}.* \
+        libpq-dev=${POSTGRES_VERSION}.* \
         libxml2-dev \
         pkg-config \
         postgresql-server-dev-${POSTGRES_VERSION}
```

Reply:
I rebased onto https://github.com/metabrainz/musicbrainz-docker/commit/af484dfc830563a917d2782b11f35451ed2e23af . I hope I did no mistake. Otherwise I will just add a fix commit.